src/openPr.js

```diff
@@ -95,7 +95,14 @@ module.exports = async function ({ context, inputs, packageVersion }) {
   const branchName = `release/${newVersion}`
 
   const messageTemplate = inputs['commit-message']
-  await execWithOutput('git', ['checkout', '-b', branchName])
+  try {
+    await execWithOutput('git', ['checkout', '-b', branchName])
+  } catch (error) {
+    core.setFailed(`${error.message}`)
+    const message = `${error.message}.  Please delete any branch with this name that may already exist and try again.`
+    logError(message)
+    throw new Error(message)
```

Comment:
I think this error handling is redundant. throwing an error and calling core.setFailed both interrupt the execution of the action. I suppose they also both log a message, meaning that explicit logging may not be necessary.

trying it out locally is not easy, but you can set up a dummy repo, point it at this branch and try to reproduce the error, and see how it fails

Reply:
You're right, it also turned out that the `checkout -b` command wasn't actually erroring as the branch didn't exist locally so I've revised my approach to check the remote before attempting to do anything else, which seems to work nicely